activemq-client/src/main/java/org/apache/activemq/util/JMSExceptionSupport.java

```diff
@@ -61,6 +63,9 @@ public static JMSException create(Exception cause) {
         if (cause instanceof JMSException) {
             return (JMSException)cause;
         }
+        if (MaxFrameSizeException.class.isAssignableFrom(cause.getClass())) {
```

Comment:
I would use instanceof here instead. isAssignableFrom is a dynamic call vs compile time so the performance is slightly worse.

Reply:
done.